### DISCUSSION TITLE
data is lost after woke up the project

### DISCUSSION THREAD

**Discussion by nickolayrusev** [nhost/nhost#1822] [Q&A]
hello all, I have strange problem... all my data is lost after I woke up the project. all users, public schemas, relations everything is new. just like starting from scratch. seems like i have blank new hasura instance? how is that possible? ideas?

**Comment by dbarrosop** [comment]
the logs should give you an idea of what might have gone wrong. In any case, ping me on discord and I can help restoring your data.

**Reply by nickolayrusev** [reply]
pinged you in discord. thanks in advance
